package com.google.android.gms.internal.ads;

import android.annotation.TargetApi;
import android.net.http.SslError;

/* JADX INFO: loaded from: classes.dex */
@TargetApi(14)
public class zzakv extends zzakt {
    @Override // com.google.android.gms.internal.ads.zzakq
    public final String zza(SslError sslError) {
        return sslError.getUrl();
    }

    @Override // com.google.android.gms.internal.ads.zzakq
    public int zzrq() {
        return 1;
    }
}
